src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -85,26 +86,26 @@ object Cluster {
              clusterUrlBase: String,
              stagingBucket: GcsBucketName): Cluster = {
     Cluster(
```

Comment:
This `Cluster.create` method is going to need to change in phase 2, since we'll be creating clusters with less information (e.g. no `stagingBucket` or `operation`). But I think it's ok to leave this for now, and refactor in phase 2.